suricata/update/main.py

```diff
@@ -937,6 +937,9 @@ def _main():
     global_parser.add_argument(
         "-c", "--config", metavar="<filename>",
         help="configuration file (default: /etc/suricata/update.yaml)")
+    global_parser.add_argument(
+        "-s", "--suricata-conf", metavar="<filename>",
```

Comment:
Looks good. But please drop the -s. There are a few things that start with s, and I don't want to dedicate one to -s yet.